rdt/transformers/datetime.py

```diff
@@ -25,13 +25,7 @@ class DatetimeTransformer(BaseTransformer):
             will be created only if there are null values. If ``True``, create the new column if
             there are null values. If ``False``, do not create the new column even if there
             are null values. Defaults to ``False``.
-        strip_constant (bool):
-            Whether to optimize the output values by finding the smallest time unit that
-            is not zero on the training datetimes and dividing the generated numerical
-            values by the value of the next smallest time unit. This, a part from reducing the
-            orders of magnitued of the transformed values, ensures that reverted values always
-            are zero on the lower time units.
-        format (str):
+        datetime_format (str):
```

Comment:
In the issue says `format` do we want to change it to `datetime_format` ?

Reply:
I don't think I can make it `format` because `format` is a special python function